mylist.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(list)
+    @list = list
+  end
```

Comment:
- [ ] When I try to test your program, I get an argument error;

![image](https://user-images.githubusercontent.com/98453979/214919027-69f8a3be-c368-4157-a895-86407a2a50d5.png)

- [ ] I updated the initialize method argument to `*list` and it worked as this allows an instance of the class to be initiated with many arguments. The method should look like this :

```sh
  def initialize(*list)
    @list = list
  end
```